gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -669,16 +683,23 @@ gst_tensor_query_client_chain (GstPad * pad,
   nns_edge_data_set_info (data_h, "client_id", val);
   g_free (val);
 
-  if (NNS_EDGE_ERROR_NONE != nns_edge_send (self->edge_h, data_h)) {
-    nns_logi ("Failed to publish to server node.");
-    goto done;
+  if (self->drop && self->requested_num >= 2) {
```

Comment:
Suggestion: new property guint 'max-request' instead 'drop'
default value 2(?) to allow sending request to server (0 for infinite), so that user can set the max number of query. :)

Reply:
Thanks, It would be better for the user. Updated.